src/cards/target_injured.ts

```diff
@@ -0,0 +1,142 @@
+import { addTarget, EffectState, ICard, Spell } from './index';
+import { CardCategory, UnitSubType } from '../types/commonTypes';
+import { Vec2, lerpVec2 } from '../jmath/Vec';
+import * as Unit from '../entity/Unit';
+import { CardRarity, probabilityMap } from '../types/commonTypes';
+import { easeOutCubic } from '../jmath/Easing';
+import * as config from '../config';
+import * as colors from '../graphics/ui/colors';
+import Underworld from '../Underworld';
+import { calculateGameDifficulty } from '../Difficulty';
+
+export const targetInjuredId = 'Target Injured';
+const healthThreshold = 20;
+const spell: Spell = {
+  card: {
+    id: targetInjuredId,
+    category: CardCategory.Targeting,
+    supportQuantity: true,
+    manaCost: 25,
+    healthCost: 0,
+    expenseScaling: 1,
+    probability: probabilityMap[CardRarity.RARE],
+    thumbnail: 'spellIconTargetInjured.png',
+    requiresFollowingCard: true,
+    description: 'spell_target_injured',
+    allowNonUnitTarget: true,
+    effect: async (state: EffectState, card: ICard, quantity: number, underworld: Underworld, prediction: boolean, outOfRange?: boolean) => {
+      // 20, 40, 80, 160, 320...
+      const adjustedHealthThreshold = healthThreshold * Math.pow(2, quantity - 1) * calculateGameDifficulty(underworld);
```

Comment:
What's the need for adjustedHealthThreshold?  Sorting them by missing health should be sufficient

Reply:
Maybe I'm misunderstanding.

This spell targets all units under a certain health threshold and adds them to the targets list in order of missing health.

Are you wanting it to instead target the next X units with the lowest hp or something?

The adjusted health threshold is just the threshold after factoring in quantity and health multipliers due to difficulty scaling (target injured would be useless in myltiplayer without this)

Reply:
Target Injured should target units in order of "most missing health"

200/300 (100 missing)
5/90 (85 missing)
120/140 (20 missing)
55/60 (5 missing)

Reply:
How would this be useless in multiplayer?

Reply:
> @jdoleary How would this be useless in multiplayer?

You can ignore that point: The spell idea was originally written/discussed as targeting things under a certain health threshold, and in that implementation multiplayer/difficulty scaling becomes a big deal, requiring multiple extra casts of the spell to have the same effect.

Health scaling isn't relevant with the implementation you're describing now, which targets the next X units with the most missing health instead of using thresholds

Sorry for the confusion 🙂 Will update the spell behavior

Reply:
New Behaviour:
- Targets the next 2 * Quantity units with the most missing HP. If multiple units have the same missing HP, it will prioritize the closest one. Ignores units at full hp.

Reply:
ah okay, I didn't remember the old description.  Yeah i think this works better :)

Is this branch ready for review?